assets/js/googlesitekit/datastore/site/info.js

```diff
@@ -648,6 +660,36 @@ export const selectors = {
 	 */
 	getUpdateCoreURL: getSiteInfoProperty( 'updateCoreURL' ),
 
+	/**
+	 * Determines if the current user can update the Plugin.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param {Object} state Data store's state.
+	 * @return {(boolean|undefined)} `true` if the current user can update the plugin, otherwise `false`.
+	 */
+	getUpdatePluginCapacity: getSiteInfoProperty( 'updatePluginCapacity' ),
+
+	/**
+	 * Determines if plugin auto updates are enabled.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param {Object} state Data store's state.
+	 * @return {(boolean|undefined)} `true` if plugin auto updates are enabled, otherwise `false`.
+	 */
+	getAutoUpdatesEnabled: getSiteInfoProperty( 'autoUpdatesEnabled' ),
+
+	/**
+	 * Get the nonce for the activate plugin auto updates ajax action.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param {Object} state Data store's state.
+	 * @return {(boolean|undefined)} `true` if plugin auto updates are enabled, otherwise `false`.
+	 */
+	getUpdatePluginNonce: getSiteInfoProperty( 'updatePluginNonce' ),
```

Comment:
Nonces are user-specific so this should be added to user-specific data and that core store.

Reply:
@aaemnnosttv where do you think it makes the most sense to place it? Which of the existing 
user stores? Or should it be placed in a new store inside the `core/user` datastore?

Reply:
@derweili how about we add it to a new partial under `core/user`? Something like `admin-ajax` would be good I think.